src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -132,23 +129,11 @@ export class CalciteValueList {
   @Event() calciteListOrderChange: EventEmitter;
 
   @Listen("calciteListItemChange") calciteListItemChangeHandler(event) {
-    const { selectedValues } = this;
-    const { value, selected, shiftPressed } = event.detail;
-    const item = event.target;
-
-    if (selected) {
-      if (!this.multiple) {
-        this.deselectSiblingItems(item);
-      }
-      if (this.multiple && shiftPressed) {
-        this.selectSiblings(item);
-      }
-      selectedValues.set(value, item);
-    } else {
-      selectedValues.delete(value);
-    }
-    this.lastSelectedItem = item;
-    this.calciteListChange.emit(selectedValues);
+    sharedListMethods.calciteListItemChangeHandler.call(this, event);
```

Comment:
Suggestion: Adding a `thisArg` to the shared methods may help clean up consuming code and also makes it clear that the method needs a different context.